src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -69,6 +71,60 @@ private void executeReport(ListingDetails listingDetails, List<SubmitEvent> subm
 
     }
 
+    private void initReport(ListingDetails listingDetails) {
+
+        var adhocReportType = new AdhocReportType();
+
+        //LocalReportsSummary fields
+        adhocReportType.setCosts(ZERO);
+        adhocReportType.setRemedy(ZERO);
+        adhocReportType.setHearingCM(ZERO);
+        adhocReportType.setHearingPrelim(ZERO);
+        adhocReportType.setReconsider(ZERO);
+        adhocReportType.setHearing(ZERO);
+        adhocReportType.setTotal(ZERO);
+        adhocReportType.setDate(ZERO);
+        adhocReportType.setSubSplit(ZERO);
+        adhocReportType.setCaseReference(ZERO);
+        adhocReportType.setLeadCase(ZERO);
+        adhocReportType.setHearingType(ZERO);
+        adhocReportType.setJudicialMediation(ZERO);
+        adhocReportType.setHearingTelConf(ZERO);
+        adhocReportType.setHearingDuration(ZERO);
+        adhocReportType.setHearingClerk(ZERO);
+
+        String hearingNumber = ZERO + "|"
```

Comment:
This should be something else...a class constant

Reply:
done